examples/common/python/connectors/direct/tcs_listener/tcs_work_order_handler.py

```diff
@@ -46,6 +50,14 @@ def __init__(self, kv_helper, max_wo_count):
 
         self.__work_order_handler_on_boot()
 
+        self.dispatcher = Dispatcher()
```

Comment:
I think this code which creates dispatcher could be move to "examples/common/python/connectors/direct/tcs_listener/tcs_listener.py"

Reply:
If the dispatcher moved into the "examples/common/python/connectors/direct/tcs_listener/tcs_listener.py", all handlers would need changing to employ this dispatcher.

As explained previously, 

> Changing all handlers at once would make it a PR too big. So rebasing other handlers in tcs_listener module would be proposed one by one later.

I would like do it step by step to ease reviewing @manojgop